docs/pages/run/getting-started/quick-start-custom-guide.md

```diff
@@ -641,8 +641,8 @@ sudo docker logs goerli-validator
 You should see something similar to:
 
 ```
-Mar-01 03:06:35.048[]                 info: Lodestar network=holesky, version=v1.16.0/6ad9740, commit=6ad9740a085574306cf46c7642e749d6ec9a4264
-Mar-01 03:06:35.050[]                 info: Connecting to LevelDB database path=/keystoresDir/validator-db-holesky
+Mar-01 03:06:35.048[]                 info: Lodestar network=hoodi, version=v1.16.0/6ad9740, commit=6ad9740a085574306cf46c7642e749d6ec9a4264
```

Comment:
having logs updated like this is a bit strange

Reply:
Prefer to actually run it successfully and update the entire new log here. Certainly v1.16.0 doesn't support hoodi.

Reply:
`count=512` with 2 local keystores as per log seems wrong but I guess we can just manually tweak those numbers

Reply:
Whoops, missed tweaking that one. I just pulled and modified these logs from our large nodes. Edited now. I think the more important thing is that they see logs that resemble these examples, which indicates things are working as it should.